xml/System.Text.Json/JsonException.xml

```diff
@@ -1,5 +1,4 @@
-﻿<Type Name="JsonException" FullName="System.Text.Json.JsonException">
-  <TypeSignature Language="C#" Value="public class JsonException : Exception" />
+﻿<Type Name="JsonException" FullName="System.Text.Json.JsonException"><TypeSignature Language="C#" Value="public class JsonException : Exception" />
```

Comment:
This change should be reverted.

Reply:
I don't know what happened here. I will revert it.

Reply:
```suggestion
<Type Name="JsonException" FullName="System.Text.Json.JsonException">
  <TypeSignature Language="C#" Value="public class JsonException : Exception" />
```